Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3954,13 +3954,16 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
             return;
 
 #if HAVE(PDFDOCUMENT_RESET_FORM_FIELDS)
-        if ([action isKindOfClass:getPDFActionResetFormClass()])
-            [m_pdfDocument resetFormFields:static_cast<PDFActionResetForm *>(action)];
+        if (RetainPtr resetAction = dynamic_objc_cast<PDFActionResetForm>(action))
+            [m_pdfDocument resetFormFields:resetAction.get()];
 #endif
 
         RetainPtr actionType = [action type];
         if ([actionType isEqualToString:@"Named"]) {
-            auto actionName = [static_cast<PDFActionNamed *>(action) name];
+            RetainPtr namedAction = dynamic_objc_cast<PDFActionNamed>(action);
```

Comment:
Can't we use `checked_objc_cast` here? The previous code seemed to assume this was the type.

Reply:
Good catch, `[actionType isEqualToString:@"Named"]` already dictates our expectation, so `checked_objc_cast` is the better option here.